packages/element/src/serialize.js

```diff
@@ -651,7 +651,7 @@ export function renderNativeComponent(
 		props = restProps;
 	} else if (
 		props.dangerouslySetInnerHTML &&
-		typeof props.dangerouslySetInnerHTML.__html === 'string'
+		props.dangerouslySetInnerHTML.__html
```

Comment:
We _could_ check `... instanceof String`, but it seems unnecessary.

Reply:
Nevermind, it wouldn't work on primitives: `( 's' ) instanceof String`

Reply:
We could check `Object.prototype.toString.call(...) === '[object String]'`, but I don't think this is necessary here.